### PR TITLE
Add lodash-webpack-plugin package

### PR DESCRIPTION
**Pull request by xprazak2**
For plugin updates, please indicate which repos this should be built into:

* [x] Nightly
* [ ] 1.16
* [ ] 1.15
* [ ] 1.14

See Foreman's [plugin maintainer documentation](http://projects.theforeman.org/projects/foreman/wiki/How_to_Create_a_Plugin#Release-strategies) for more information.

---

### CODE REVIEW

File: nodejs-lodash-webpack-plugin/nodejs-bundle-lodash-webpack-plugin.spec [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+%define npm_cache_dir /tmp/npm_cache_%{name}-%{version}-%{release}
+
+%description
+%{summary}
+
+%prep
+mkdir -p %{npm_cache_dir}
+for tgz in %{sources}; do
+  echo $tgz | grep -q registry.npmjs.org || npm cache add --cache ./%{npm_cache_dir} $tgz
```
Comment (ekohl) [inline nodejs-lodash-webpack-plugin/nodejs-bundle-lodash-webpack-plugin.spec]: This shouldn't be ```./%(npm_cache_dir}``` but just ```%{npm_cache_dir}``` I think

File: nodejs-lodash-webpack-plugin/nodejs-bundle-lodash-webpack-plugin.spec [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+
+%files
+%{nodejs_sitelib}/%{npm_name}
+
+%license LICENSE
+%doc README.md
+
+%changelog
+* Tue Jan 2 2018 oprazak@redhat.com <Ondrej Prazak> - 0.11.4-1
```
Comment (ekohl) [inline nodejs-lodash-webpack-plugin/nodejs-bundle-lodash-webpack-plugin.spec]: We add these while tagging

**Comment by xprazak2** [comment]
Updated with changes

File: nodejs-lodash-webpack-plugin/nodejs-bundle-lodash-webpack-plugin.spec [hunk 3]
```diff
@@ -0,0 +1,55 @@
+%global npm_name lodash-webpack-plugin
+
+%{?nodejs_find_provides_and_requires}
```
Comment (ekohl) [inline nodejs-lodash-webpack-plugin/nodejs-bundle-lodash-webpack-plugin.spec]: We don't add this anymore

**Comment by xprazak2** [comment]
I renamed the spec and removed `%{?nodejs_find_provides_and_requires}`

File: nodejs-lodash-webpack-plugin/nodejs-lodash-webpack-plugin.spec [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+%prep
+mkdir -p %{npm_cache_dir}
+for tgz in %{sources}; do
+  echo $tgz | grep -q registry.npmjs.org || npm cache add --cache %{npm_cache_dir} $tgz
+done
+
+%setup -T -q -a 2 -D -n %{npm_cache_dir}
+
+%build
+npm install --cache-min Infinity --cache . --no-optional --global-style true %{npm_name}@%{version}
```
Comment (ekohl) [inline nodejs-lodash-webpack-plugin/nodejs-lodash-webpack-plugin.spec]: the . here needs to be ```%{npm_cache_dir}``` too.

**Comment by ekohl** [comment]
@dLobatog could you have another look?

**Pull request merged**